pkg/proxy/proxy.go

```diff
@@ -269,10 +270,31 @@ func parseIPPort(ipstr string, info *accesslog.EndpointInfo) {
 	}
 }
 
-func (r *Redirect) getSourceInfo(req *http.Request) (accesslog.EndpointInfo, accesslog.IPVersion) {
+/*
+   This function fills the accesslog.EndpointInfo fields, by fetching the consumable from the consumable cache
+   of endpoint using identity sent by source. This is needed in ingress proxy while logging the source endpoint info.
+   Since there will be 2 proxies on the same host, if both egress and ingress policies are set, the ingress policy cannot determine the
+   source endpoint info based on ip adress, as the ip address would be that of the egress proxy i.e host.
+*/
+func fillInfoFromConsumable(ipstr string, info *accesslog.EndpointInfo, srcIdentity policy.NumericIdentity, ep ProxySource) {
```

Comment:
small nit: any reason why this function is not a method of `accesslog.EndpointInfo`?